### PR TITLE
[doc] Enablement Ops feedback

### PR DESCRIPTION
**Pull request by ddhodge**
Feedback and tidyups

@netlify /preview/yugabyte-platform/install-yugabyte-platform/install-software/installer/

### CODE REVIEW

**Comment by kerneljake** [comment]
LGTM!

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ type: docs
...
+You can configure YugabyteDB Anywhere to use OpenID Connect (OIDC) for single sign-on (SSO) authentication to access to your YugabyteDB Anywhere instance.
 
-YugabyteDB Anywhere uses OIDC to enable single sign-on (SSO) authentication for access to your YugabyteDB Anywhere instance.
+OIDC is an authentication protocol that allows client applications to confirm the user's identity via authentication by an authorization server.
 
-For information on configuring a YugabyteDB Anywhere universe to use OIDC-based authentication for YugabyteDB YSQL database access using Microsoft Entra (Azure AAD) as the Identity Provider (IdP), refer to [OIDC authentication with Microsoft Entra](../../security/authentication/oidc-authentication-aad/).
+(For information on configuring a YugabyteDB Anywhere universe to use OIDC-based authentication for YugabyteDB YSQL database access using Microsoft Entra (Azure AAD) as the Identity Provider (IdP), refer to [OIDC authentication with Microsoft Entra](../../security/authentication/oidc-authentication-aad/).)
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md:34]: why these in brackets?

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md [hunk 2]
```diff
@@ -98,7 +98,12 @@ Navigate to **Configs > Infrastructure > Amazon Web Services** to see a list of
 
 To view a provider, select it in the list of AWS Configs to display the **Overview**.
 
-To edit the provider, select **Config Details**, make changes, and click **Apply Changes**. For more information, refer to [Provider settings](#provider-settings). Note that, depending on whether the provider has been used to create a universe, you can only edit a subset of options.
+To edit the provider, select **Config Details**, make changes, and click **Apply Changes**. For more information, refer to [Provider settings](#provider-settings). Note that for YBA version 2.20.1 and later, if the provider has been used to create a universe, you can only edit a subset of fields, including the following:
+
+- Provider Name
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:103]: So this list of options were supported only for 2.18.5 and 2.20.1.

File: docs/content/preview/yugabyte-platform/security/authentication/_index.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
-title: YugabyteDB Anywhere Authentication
-headerTitle: Authentication
-linkTitle: Authentication
+title: Configure database authentication in YugabyteDB Anywhere
+headerTitle: Database authentication
+linkTitle: Database authentication
 headcontent: Secure your YugabyteDB universes
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/security/authentication/_index.md:5]:
```suggestion
headcontent: Secure your YugabyteDB universes in YugabyteDB Anywhere
```

**Review by aishwarya24 — APPROVED** [review]
minor comments

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md:34]: I felt they were parenthetical to the matter at hand.

**Pull request merged**